src/swordxx/versificationmgr.cpp

```diff
@@ -94,14 +94,17 @@ class VersificationMgr::System::Private {
         verse of another chapter in default intermediate canon(kjva), so mapping data
         contains expections. Intermediate canon could not contain corresponding data.
 
-        pointers on uchar[7]: 1 value - book id 1-based, ot+nt, 2-4 map to, 5-7 map
-        from (chap,verse from, verse to if greater then "verse from")
-
-        TODO what if book name in one v11n differs from cannon
-            special section in mapping for book transformation
+        Each element in @variable mappings contains of all rules that are related to
+        particular book.
+
+        @typedef mappingRule is a pointer on uchar[7]: 1 value - book id 1-based, ot+nt,
+        2-4 map to, 5-7 map from (chap,verse from, verse to if greater then "verse from").
+        Size of rule would be 8 if there is inter-book mapping and target book is abscent
+        in reference system, in this case @variable mappingsExtraBooks data is used and
+        id is rule book id minus book count.
     */
```

Comment:
I suggest that the `uchar[7]` be documented using a list and with a bit more verbosity. To be frank, I don't understand what is going on here, and this comment is not very useful to me. I must take some time to try to understand.